Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -155,6 +155,39 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 }
                 return scope
             }
+            
+            options.configureUserFeedback = { config in
```

Comment:
SHould we call this: `configureUserFeedbackWidget` ? might be too verobse but we're only configuring the widget behavior here, right? Not `captureUserFeedback` behavior?

Reply:
It's more than just the widget, there is also the form open/close hooks, success/error hooks, whether to use the configured sentry user from scope... we could call it `configureUserFeedbackIntegration`?

Reply:
tbqh I don't see the value in appending `Integration` to the name. We do have the `Integration` properties on options but even Integration 'types' I don't think needs appending `Integration`.
In that case `configureUserFeedback` is a better choice imo. 

Since we're renaming the func from `captureUserFeedback` to `captureFeedback`, how about `configureFeedback` ? Or `configureFeedbackUI` if options under that are all related to the behavior of the UI components.

We need to be clear that the success/failure hooks won't be called if you simply called `captureFeedback`, right? That's just when using the modal. At least on JS that's the way afaik

Reply:
> Since we're renaming the func from `captureUserFeedback` to `captureFeedback`

I forget where we had this conversation, but I'm not sure it's actually necessary. For now, I've added notes in the headerdoc of `SentrySDK.captureUserFeedback` (the current one) and `SentrySDK.showUserFeedbackForm` (the new one that immediately pops up the new form) as well as `SentryOptions.configureUserFeedback` itself. See if you think those are sufficient, and if not, we can brainstorm more on a better name for `configureUserFeedback`.

Yes, the hooks are not called with the preexisting function. Only with our UI form. I made a note of it in the headerdocs for `onSubmitSuccess`/`onSubmitError`.

Reply:
If we plan to rename `captureUserFeedback` to `captureFeedback`, I think we should remove `user` from the naming here to be consistent. 

> It's more than just the widget, there is also the form open/close hooks, success/error hooks, whether to use the configured sentry user from scope

But still, all that functionality could be part of the widget. For me, `configureFeedbackWidget` sounds like a reasonable name. We could rename the `SentryUserFeedbackWidgetConfiguration` to `SentryFeedbackTriggerButtonConfiguration` or something similar to avoid the naming collision. The `SentryUserFeedbackWidgetConfiguration` is only a button to trigger showing the widget.

Reply:
> The SentryUserFeedbackWidgetConfiguration is only a button to trigger showing the widget

This confuses what is a widget and what is the form. 

The button that triggers the form to display is the widget.

Reply:
I leave it up to you to make the best choice then for the API. I guess we will do an experimental release first, so we can still change it if we get some feedback. Also, this isn't an essential API of our SDK. You most likely set it up once and then leave it. Still, it shouldn't be confusing.

Reply:
On the web SDK we vaguely refer to the button + form as the widget. For the button, we call it `actor` in some parts of the code, and publicly [here](https://docs.sentry.io/platforms/javascript/user-feedback/configuration/) we call it `trigger`

Reply:
`actor`, `widget`, `button`, `form`, `modal` and `trigger` all have specific meanings in the Apple Human Interface Guidelines and Developer documentation that our customers will already be familiar with. We should avoid overloading or blurring the lines between them, and instead use the appropriate terms to convey what each piece of this feature will do.

- [`trigger`](https://developer.apple.com/documentation/swiftui/primitivebuttonstyleconfiguration/trigger()): performs [a] button’s action ([also](https://developer.apple.com/documentation/realitykit/creating-a-trigger): Define when and how a behavior fires) 
- [`actor`](https://developer.apple.com/documentation/swift/actor): actors execute tasks on a shared global concurrency thread pool
- [`button`](https://developer.apple.com/design/human-interface-guidelines/buttons): initiates an instantaneous action ([also](https://developer.apple.com/documentation/swiftui/button): A control that initiates an action)
- [`widget`](https://developer.apple.com/design/human-interface-guidelines/widgets): People tap or click a widget to launch its corresponding app (in our case, we aren't launching an app as we're an SDK inside an app–we're displaying a modal screen inside that app)
- [`form`](https://developer.apple.com/documentation/swiftui/form): A container for grouping controls used for data entry
- [`modal`](https://developer.apple.com/design/Human-Interface-Guidelines/modality): Modality is a design technique that presents content in a separate, dedicated mode that prevents interaction with the parent view and requires an explicit action to dismiss

The way I would describe this feature:
> The User Feedback **integration** embeds a **widget** in your application that, when pressed, displays a **modal** **form** for your end user to fill in details of their experience that will help you become aware of and fix issues, and to generally improve your product.

Implicitly, the **widget** is (or at least contains) a **button** (although UIButton has always been able to contain both a textual and image component, which is all our widget UI has), and that button's **trigger** is opening the **form**.

I would completely avoid using the term "actor" when describing this feature. I also don't see that term at all on the docs page. For comparison, here is the first description of how the feature works from our [Javascript doc](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-widget): 
> The embeddable JavaScript widget allows users to submit feedback from anywhere inside your application. The [Crash-Report Modal](https://docs.sentry.io/#crash-report-modal) collects reactive feedback tied to an error event

On the topic of not wanting to append "integration" onto the end of this options property to call it `configureFeedbackIntegration`, note that's essentially what the [Javascript SDK](https://docs.sentry.io/platforms/javascript/user-feedback/configuration/#general) does:
> The following options can be configured for the integration in feedbackIntegration({})